### PR TITLE
fix(content): #852 Only show one preview on new tab

### PR DESCRIPTION
**Pull request by k88hudson**
This change is [Reviewable](https://reviewable.io/reviews/mozilla/activity-stream/921)

### CODE REVIEW

File: content-src/selectors/selectors.js [hunk 1]
```diff
@@ -148,8 +148,10 @@ module.exports.selectSitePreview = createSelector(
       thumbnail = getBestImage(site.images);
       if (type === "video") {
         previewURL = getVideoPreview(site.url);
+        console.log("previewURL", site.url, previewURL);
```
Comment (sarracini) [inline content-src/selectors/selectors.js]: did you mean to leave this console.log here?

File: content-src/components/ActivityFeed/ActivityFeed.js [hunk 2]
```diff
@@ -32,7 +32,8 @@ const ActivityFeedItem = React.createClass({
     return {
       onShare: function() {},
       onClick: function() {},
-      showDate: false
+      showDate: false,
+      showPreview: true
```
Comment (sarracini) [inline content-src/components/ActivityFeed/ActivityFeed.js]: I can't find where this is used... did you mean to do:
```<MediaPreview previewInfo={this.props.showPreview}``` maybe?

File: content-test/components/ActivityFeed.test.js [hunk 3]
```diff
@@ -158,6 +159,30 @@ describe("GroupedActivityFeed", function() {
...
+      const feed = renderWithProvider(<GroupedActivityFeed sites={sites} />);
+      const previews = TestUtils.scryRenderedComponentsWithType(feed, MediaPreview);
+      assert.lengthOf(previews, sites.length, "one preview per site");
+    });
+    it("should create no previews if maxPreviews=0", () => {
+      const feed = renderWithProvider(<GroupedActivityFeed sites={sites} maxPreviews={0} />);
+      const previews = TestUtils.scryRenderedComponentsWithType(feed, MediaPreview);
+      assert.lengthOf(previews, 0, "no previews");
+    });
+    it("should create previews for n sites if maxPreviews=n", () => {
```
Comment (sarracini) [inline content-test/components/ActivityFeed.test.js]: 👍 👍 👍  yessss i like tests that can be changed easily based on new requirements

File: content-src/selectors/selectors.js (hunk 1 above)
Comment (k88hudson) [inline content-src/selectors/selectors.js]: 😮

File: content-src/components/ActivityFeed/ActivityFeed.js (hunk 2 above)
Comment (k88hudson) [inline content-src/components/ActivityFeed/ActivityFeed.js]: not sure how this got in here tbh

File: content-test/components/ActivityFeed.test.js (hunk 3 above)
Comment (k88hudson) [inline content-test/components/ActivityFeed.test.js]: 😎

**Comment by sarracini** [comment]
R+ 🎈

**Pull request merged**